tests/CMakeLists.txt

```diff
@@ -1,21 +1,30 @@
-include_directories("../include")
-include_directories("../examples/common")
-include_directories("${PROJECT_BINARY_DIR}/generated")
+set(sc2test_sources
+    all_tests.cc
+    feature_layers_shared.cc
+    test_ability_remap.cc
+    test_actions.cc
+    test_app.cc
+    test_feature_layer_mp.cc
+    test_feature_layer.cc
+    test_framework.cc
+    test_movement_combat.cc
+    test_multiplayer.cc
+    test_observation_interface.cc
+    test_performance.cc
+    test_rendered.cc
```

Comment:
I believe `test_rendered.cc` shouldn't be included here if we are compiling disabled `BUILD_SC2_RENDERER`, but the original code has this issue too.